test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RegistryKeyUtilityTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Win32;
+using NuGet.Test.Utility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NuGet.CommandLine.Test
+{
+    public class RegistryKeyUtilityTests
+    {
+        private readonly TestLogger _logger;
+
+        public RegistryKeyUtilityTests(ITestOutputHelper output)
+        {
+            _logger = new TestLogger(output);
+        }
+
+        [WindowsNTFact]
+        public void GetValueFromRegistryKey_UnknownKey_ReturnsNull()
+        {
+            // Prepare
+            var guidString = new Guid().ToString();
+            var registryKey = RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Default);
+
+            // Act
+            var value = RegistryKeyUtility.GetValueFromRegistryKey(guidString, "key_local_machine\\test\\folder", registryKey, _logger);
+
+            // Assert
+            Assert.Null(value);
+            Assert.Empty(_logger.Messages);
+        }
+
+        [WindowsNTFact]
+        public void GetValueFromRegistryKey_GetSampleKey_ReturnsNotNull()
+        {
+            // Act
+            var value = RegistryKeyUtility.GetValueFromRegistryKey("ProductName", @"SOFTWARE\Microsoft\Windows NT\CurrentVersion", Registry.LocalMachine, _logger);
```

Comment:
Any reason you're using the string literal with "@" here and on line 29 you're escaping the slashes? Should you use string literal in both or escape in both for consistency?

Reply:
No reason at all. Changed to literal string.